src/components/tts.js

```diff
@@ -76,7 +76,17 @@ class MiniMediaPlayerTts extends LitElement {
         message,
         ...config.data,
       });
-    else this.hass.callService('tts', `${config.platform}_say`, opts);
+    else if (config.platform === 'service') {
+      const [domain, service] = (config.data.service || '').split('.');
+      const field = config.data.message_filed || 'message';
+      const serviceData = {
+        [field]: message,
+        entity_id: opts.entity_id,
+        ...(config.language ? { language: opts.language } : {}),
+        ...(config.data.service_data || {}),
+      };
+      this.hass.callService(domain, service, serviceData);
+    } else this.hass.callService('tts', `${config.platform}_say`, opts);
```

Comment:
It's probably time to move from `if else` cases to a `switch` here, imo.
Nothing you have to do in this PR though if you don't feel like it.

Reply:
I feel good. 😄